exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/DatabaseMigrationTests.kt

```diff
@@ -132,6 +133,65 @@ class DatabaseMigrationTests : DatabaseTestsBase() {
         }
     }
 
+    @Test
+    fun testDropUnmappedColumnsStatementsIdentical() {
+        val t1 = object : Table("foo") {
+            val id = integer("idcol")
+
+            override val primaryKey = PrimaryKey(id)
+        }
+
+        val t2 = object : Table("foo") {
+            val id = integer("idcol")
+
+            override val primaryKey = PrimaryKey(id)
+        }
+
+        withTables(t1) {
+            val statements = MigrationUtils.dropUnmappedColumnsStatements(t2, withLogs = false)
+            assertEqualCollections(statements, emptyList())
+        }
+    }
+
+    @Test
+    fun testDropUnmappedColumnsStatementsIdentical2() {
+        val t1 = object : Table("foo") {
+            val id = integer("idCol")
```

Comment:
Is the only difference between this test and the one above the 'C' or is this an accidental duplicate?
If column casing/detection is important, could we maybe instead have a single test with identical tables that have 3 columns each, one for each way to name columns?. Meaning anything like: `"col1"`, `"cOL2"`, and `"\"Col3\""`.

Reply:
It's not accidental. I copied them from `CreateMissingTablesAndColumnsTests`.
Good idea 👍🏼